pkg/apis/eksctl.io/v1alpha5/validation.go

```diff
@@ -270,7 +270,6 @@ func (c *ClusterConfig) unsupportedVPCCNIAddonVersion() (bool, error) {
 	for _, addon := range c.Addons {
 		if addon.Name == VPCCNIAddon {
 			if addon.Version == "" {
-				addon.Version = minimumVPCCNIVersionForIPv6
```

Comment:
so.... if it's empty we leave it empty?

Reply:
yep. Default value means the EKS API uses the default value. E.g.:
```
 eksctl utils describe-addon-versions --name vpc-cni --kubernetes-version 1.22
2022-04-06 15:22:57 [ℹ]  eksctl version 0.88.0
2022-04-06 15:22:57 [ℹ]  using region us-west-2
2022-04-06 15:22:57 [ℹ]  describing addon versions for addon: vpc-cni
{
  Addons: [{
      AddonName: "vpc-cni",
      AddonVersions: [
        {
          AddonVersion: "v1.10.2-eksbuild.1",
          Architecture: ["amd64","arm64"],
          Compatibilities: [{
              ClusterVersion: "1.22",
              DefaultVersion: false,
              PlatformVersions: ["*"]
            }]
        },
        {
          AddonVersion: "v1.10.1-eksbuild.1",
          Architecture: ["amd64","arm64"],
          Compatibilities: [{
              ClusterVersion: "1.22",
              DefaultVersion: true,
              PlatformVersions: ["*"]
            }]
        },
        {
          AddonVersion: "v1.9.3-eksbuild.1",
          Architecture: ["amd64","arm64"],
          Compatibilities: [{
              ClusterVersion: "1.22",
              DefaultVersion: false,
              PlatformVersions: ["*"]
            }]
        },
        {
```

`v1.10.1-eksbuild.1` would be used